stable/spring-cloud-data-flow/values.yaml

```diff
@@ -64,6 +64,7 @@ deployer:
     initialDelaySeconds: 120
   livenessProbe:
     initialDelaySeconds: 90
+  imagePullPolicy: IfNotPresent
```

Comment:
deployer properties should start being split out so they can be customized on stream/batch basis, so with the addition of `imagePullPolicy` to `server-config`, something like:

```
batch:
  imagePullPolicy: ..
streaming:
 imagePullPolicy: ..
```

the naming would be in line with the `features` block

then reference `deployer.(batch|streaming).imagePullPolicy...` in the appropriate location